app/controllers/casa_admins_controller.rb

```diff
@@ -1,16 +1,19 @@
 class CasaAdminsController < ApplicationController
-  before_action :authenticate_user!, :must_be_admin
   before_action :set_admin, except: [:index, :new, :create]
   before_action :require_organization!
+  after_action :verify_authorized
 
   def index
+    authorize CasaAdmin
     @admins = policy_scope(current_organization.casa_admins)
   end
 
   def edit
+    authorize @casa_admin, policy_class: CasaAdminPolicy
```

Comment:
Just curious why you explicitly added the `policy_class` here -- shouldn't Pundit be able to infer it?

Reply:
Oh, I see, it is probably using the UserPolicy by default. Never mind!

Reply:
I thought it had to do with the polymorphic associations of users. Although after poking around a bit a more I see a method in the `User` model that is probably causing this. I'll delete that and see if I can delete all these `policy_class`s. Thanks!

```ruby
  def policy_class
    case type
    when Volunteer
      VolunteerPolicy
    else
      UserPolicy
    end
  end
```